src/codemagic/tools/_app_store_connect/action_groups/review_submissions_actions_group.py

```diff
@@ -76,6 +81,33 @@ def cancel_review_submission(
             canceled=True,
         )
 
+    @cli.action(
+        'cancel-review-submissions',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        AppStoreVersionArgument.PLATFORM,
+        ReviewSubmissionArgument.REVIEW_SUBMISSION_STATE,
+        action_group=AppStoreConnectActionGroup.REVIEW_SUBMISSIONS,
+    )
+    def cancel_review_submissions(
+        self,
+        application_id: ResourceId,
+        platform: Optional[Platform] = None,
+        review_submission_state: Optional[Union[ReviewSubmissionState, Sequence[ReviewSubmissionState]]] = None,
+        should_print: bool = False,
+    ) -> List[ReviewSubmission]:
+        """
+        Find and cancel review submissions in App Store Connect
```

Comment:
Shouldn't the docstring indicate that it cancels review submissions only for specified application?